website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,235 @@
+
+All the tools defined in Tools Definitions can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
+
+
+![](assets/applications/tools_filters_profiles.png)
+
+1. **Folder Paths:** Create a list of folder paths to determine which ones the filter profile will affect. *It supports regex patterns.*
+2. **Task Types:** Choose from a list of task types to determine which ones the filter profile will affect.
+3. **Task Names:** Create a list of task names to determine which ones the filter profile will affect.
+4. **Tools:** Tools list to determine which ones the filter profile will affect.
+5. **+ button:** To add more applications filters profiles.
+
+:::tip Example Filter profile
+Find example of Tool defintion and a Filter profile in the FAQ section below:
+[How to make a tool available for all folder paths in a project for a specific application variant ?](addon_applications_admin#how-to-make-a-tool-available-for-all-folder-paths-in-a-project-for-a-specific-application-variant-)
+:::
+
+## FAQ
+### How to use application variant group settings ?
+
+<details><summary>How to use application variant group settings ?</summary>
+
+Variant group settings is used to add more versions of an application.
+There are two ways of doing it.
+
+1. **Add new executable** to an existing application version. This is a good way if you have multiple fully compatible versions of your DCC across the studio. Nuke is a typical example where multiple artists might have different `v#` releases of the same minor Nuke release. For example `12.2v3` and `12.3v6`. When you add both to `12.2` Nuke executables they will be treated the same in AYON and the system will automatically pick the first that it finds on an artist machine when launching. Their order is also the order of their priority when choosing which version to run if multiple are present.
+![settings_applications](assets/applications/settings_addapplication.gif)
+
+2. **Add version** in case you want this version to be selectable individually. This is typically used for bigger releases that might not be fully compatible with previous versions. Keep in mind that if you add the latest version of an Application that is not yet part of the official AYON release, you might run into problems with integration. We test all the new software versions for compatibility and most often, smaller or bigger updates to AYON code are necessary to keep everything running.
+![settings_applications](assets/applications/settings_addappversion.gif)
+
+</details>
+
+### What is host name used for ?
+
+<details><summary>What is host name used for ?</summary>
+
+AYON uses host name to match applications to their host integration addons.
```

Comment:
If this would stay here I would relate it to applications addon, not to AYON.
```suggestion
Applications addon uses host name to match applications to their host integration addons.
```

Reply:
then. I'll fix it and then we can figure out where to put it.